index.js

```diff
@@ -316,6 +316,10 @@ function HttpRequest(options) {
 			}
 		};
 	}
+
+	if ('onerror' in xhr && !xhr.onreadystatechange) {
+		xhr.onerror = onLoad;
```

Comment:
Ah yes, CORS in IE:
https://developer.mozilla.org/en-US/docs/Web/API/XDomainRequest
Note how that only has an onload event and no readystatechange.